components/datetime/tests/de.rs

```diff
@@ -0,0 +1,8 @@
+use icu_datetime::provider::*;
+
+#[test]
+fn de() {
```

Comment:
`suggestion`: can you pick a more descriptive name for the test? It shows up in test list and should be possible to filter for with `cargo test X`.

Reply:
I don't intend to check this in; I opened this draft PR to demonstrate an issue I'm trying to debug that I didn't know if you'd have any thoughts on.